map/lanelet2_extension/lib/visualization.cpp

```diff
@@ -803,6 +803,11 @@ visualization_msgs::msg::MarkerArray visualization::generateLaneletIdMarker(
   const lanelet::ConstLanelets road_lanelets, const std_msgs::msg::ColorRGBA c, const double scale)
 {
   visualization_msgs::msg::MarkerArray markers;
+  if (road_lanelets.empty()) {
+    RCLCPP_WARN_STREAM(
+      rclcpp::get_logger("lanelet2_extension.query"),
+      "No road found in lanelet map. Thus vector map will not appear on Rviz.");
```

Comment:
I think it would make more sense if you modify lanelet2_map_visualizer node rather than modifying the lanelet2_extension library since library can be used for different purpose.

For example somewhere around [here](https://github.com/autowarefoundation/autoware.universe/blob/1a011896bb17796099a63cf3da859e4e142513e4/map/map_loader/src/lanelet2_map_loader/lanelet2_map_visualization_node.cpp#L94).

Reply:
Agree, thank you!
Fixed by https://github.com/autowarefoundation/autoware.universe/pull/1218/commits/015be4b944b3005d32b9cd8a39fa2f1da59a7ce5.